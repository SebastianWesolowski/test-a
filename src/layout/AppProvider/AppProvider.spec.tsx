import renderer from "react-test-renderer";
import "@testing-library/jest-dom";

import { AppProvider } from ".";

describe("Components", () => {
  describe("AppProvider", () => {
    const component = renderer.create(<AppProvider />);
    const tree = component.toJSON();

    it("should render without errors", () => {
      expect(tree).toMatchSnapshot();
    });
  });
});
