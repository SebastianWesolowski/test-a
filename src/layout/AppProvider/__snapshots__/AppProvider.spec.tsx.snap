// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components AppProvider should render without errors 1`] = `
<div
  className=""
/>
`;
